nengo/utils/numpy.py

```diff
@@ -255,3 +255,86 @@ def rmse(x, y, axis=None, keepdims=False):  # pragma: no cover
 
     def rfftfreq(n, d=1.0):
         return np.abs(np.fft.fftfreq(n=n, d=d)[: n // 2 + 1])
+
+
+_betaincinv22_file = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)), "betaincinv22_table.npz"
```

Comment:
Is there a reason to prefer the location `nengo/utils` over `nengo-data` or vice-versa?

Reply:
:shrug: @tbekolay?

Reply:
Trevor said `nengo/utils` is good, because we're going to retire `nengo-data` at some point. I've added it to the manifest and tested that it gets installed with `python setup.py install` (this required `include_package_data=True` in `setup.py`).